dist/cows/C3PO.cow

```diff
@@ -0,0 +1,20 @@
+# C3PO
```

Comment:
The dist folder shouldn't be pushed to the repo

Reply:
dist folder included in the .gitignore

Reply:
Yes, but you need to delete it from the repo now

Reply:
Done